README.md

```diff
@@ -55,6 +55,8 @@ search these unofficial resources:
 * [Questions tagged 'node.js' on StackOverflow][]
 * [#node.js channel on chat.freenode.net][]. See <http://nodeirc.info/> for more
   information.
+* [Node.js Slack Community](http://node-js.slack.com): Visit
+  [nodeslackers.com](http://www.nodeslackers.com/) to register.
```

Comment:
I wish this URL was https. Looks like it's not served over HTTPS unfortunately. Leaving this comment here in case whoever runs it sees this and there's anything that can be done about it. Non-blocking on landing this.

Reply:
😬 woops, yeah i'll get on that first thing tomorrow and let you know